File: semester-5/scripting-langs/lab6/Task2.java
import java.io.File;
import java.util.Scanner; 

class Task2 {
	public static void main(String[] args) {
		try {
			File covidFile = new File("Covid.txt");
			Scanner fileReader = new Scanner(covidFile);
			int totalCases = 0;

			fileReader.nextLine();
			while (fileReader.hasNextLine()) {
				String line = fileReader.nextLine();
				
				totalCases += Integer.parseInt(line.split("\t")[4]);
			}
			fileReader.close();

			System.out.println(totalCases);
		} catch (Exception e) {
			System.out.println("An error occurred.");
			e.printStackTrace();
		}
	}
}

File: semester-5/scripting-langs/lab6/task2.py
import time

FILE_PATH = 'Covid.txt'

def calc_cases(file_name):
  with open(file_name, 'r') as covid_file:
    covid_file.readline() # skip header

    cases = 0
    for line in covid_file.readlines():
      row = line.split('\t')
      try:
        cases += int(row[4])
      except:
        pass
 
    return cases

try:
  print(calc_cases(FILE_PATH))
except:
  print('No Covid.txt file')